melosys-kodeverk-java/melosys-kodeverk-generator/src/test/resources/dist/kodemap.yml

```diff
@@ -39,6 +39,35 @@ avklartefaktatyper:
     term: null
   - kode: ARBEIDSGIVERS_FORRETNINGSSTED
     term: null
+avtaleland:
+  - kode: AU
+    term: Australia
+  - kode: BA
+    term: Bosnia-Hercegovina
+  - kode: CA
+    term: Canada
+  - kode: CL
+    term: Chile
+  - kode: IN
+    term: India
+  - kode: IL
+    term: Israel
+  - kode: ME
+    term: Montenegro
+  - kode: RS
+    term: Serbia
+  - kode: GB
+    term: Storbritannia
```

Comment:
GB eller UK? Tenker bare mtp TRYGDEAVTALE_UK 🙃

Reply:
Landkoden er GB selv om det strengt tatt er UK ettersom det omfatter Nord-Irland og de to andre som nevnt over.

Reply:
Nord-Irland har forøvrig ikke egen landkode